.fcrepo_wrapper

```diff
@@ -0,0 +1,10 @@
+# Place any default configuration for fcrepo_wrapper here
+port: 8984
+enable_jms: false
+
+download_dir: tmp/fcrepo_wrapper
+
+fcrepo_home_dir: tmp/fcrepo4-development-data
+
+version: 4.7.3-RC-1
```

Comment:
I don't think we want `master` Hyrax on the bleeding edge quite yet. I'd prefer if folks specify versions themselves, or if we pull the latest version allowed by `fcrepo_wrapper`. This is going to be a pain to remember to update, too.